src/styles/blocks/reset.css

```diff
@@ -0,0 +1,7 @@
+html {
+  font-family: Roboto, sans-serif;
+}
```

Comment:
it's more about normalizing css (as an option, u can name this file looks like 'general.css')